db/schema.rb

```diff
@@ -10,25 +10,25 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_09_09_180131) do
+ActiveRecord::Schema[7.0].define(version: 2023_09_10_204248) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "saved_recipies", force: :cascade do |t|
+  create_table "saved_recipes", force: :cascade do |t|
     t.bigint "user_id", null: false
-    t.integer "api_recipie_id", null: false
+    t.integer "api_recipe_id", null: false
     t.boolean "favorited", default: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.index ["user_id"], name: "index_saved_recipies_on_user_id"
+    t.index ["user_id"], name: "index_saved_recipes_on_user_id"
   end
 
-  create_table "scheduled_recipies", force: :cascade do |t|
-    t.bigint "saved_recipie_id", null: false
+  create_table "scheduled_recipes", force: :cascade do |t|
+    t.bigint "saved_recipe_id", null: false
     t.datetime "scheduled_date"
```

Comment:
What are your thoughts on using Date vs Datetime? Do you think we should keep this nullable? I have no clue which would be better. I can see pros/cons to both.

Reply:
I think datetime makes more sense in this scenario in case we want to add a feature where a user can choose a time of the day as well as which day to schedule a recipe.